src/test/java/com/basho/riak/client/core/util/DefaultCharsetTest.java

```diff
@@ -0,0 +1,49 @@
+package com.basho.riak.client.core.util;
+
+import java.nio.charset.Charset;
+
+import org.junit.Test;
+import static org.junit.Assert.*;
+
+public class DefaultCharsetTest
+{
```

Comment:
Excuse me for misleading comment, my point was that it may worth to create corresponding test case to fix the desired behavior in it. If test passed, we have an expected behavior.